test/aws_cycles.go

```diff
@@ -191,8 +192,22 @@ func GetItemAppReleaseCycle(appName string) awsutil.Cycle {
 }
 
 func UpdateAppStackCycle(stackName, count, memory string) awsutil.Cycle {
+	const countPreviousValue string = "1"
```

Comment:
Small go feedback. This could be something simpler like `previousCount := "1"`, and I think in general with Go simpler code wins.